apps/commudle-admin/src/app/feature-modules/users/components/public-profile/user-extra-details/user-badges/user-badges.component.html

```diff
@@ -1,6 +1,6 @@
 <nb-card *ngIf="badges.length > 0" class="user-badges" id="badges">
   <nb-card-header>
-    <nb-icon icon="award" status="warning"></nb-icon>
+    <fa-icon [icon]="faAward" class="com-text-[#ffaa00]"></fa-icon>
```

Comment:
Discuss with @Ashwin-op on why this color code is not working from tailwind list of colors